code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -457,6 +457,13 @@
 	race = /datum/species/jelly/slime
 	mutationtext = "<span class='danger'>The pain subsides. Your whole body feels like slime.</span>"
 
+/datum/reagent/mutationtoxin/felinid
+	name = "Felinid Mutation Toxin"
+	id = "felenidmutationtoxin"
```

Comment:
felinid

Reply:
That's what happens when you make shitpost additions at 3am I guess.